roles/kubernetes/master/defaults/main/main.yml

```diff
@@ -44,6 +44,9 @@ discovery_timeout: 5m0s
 # Instruct first master to refresh kubeadm token
 kubeadm_refresh_token: true
 
+# Scale down coredns replicas to 0 if using not using coredns dns_mode
```

Comment:
`if using not using coredns dns_mode`
I guess it's "if not using coredns dns_mode"

Reply:
oh thanks for catching the bad wording